skimage/segmentation/tests/test_random_walker.py

```diff
@@ -582,3 +582,11 @@ def test_umfpack_import():
     except ImportError:
         assert UmfpackContext is None
     return
+
+
+def test_empty_labels():
+    from skimage.segmentation import random_walker
+    image = np.random.random((5, 5))
+    labels = np.zeros((5, 5), dtype=int)
+    with testing.raises(ValueError):
```

Comment:
The original error is also a ValueError, so we must actually look at the error message.

```suggestion
    with testing.raises(ValueError, match="No seeds provided"):
```